docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -50,9 +50,15 @@ Vivarium - CSU - Simulating Multiple Myeloma registries
 | MM    | Multiple myeloma           |
 +-------+----------------------------+
 | OS    | Overall Survival           |
+<<<<<<< HEAD
```

Comment:
You need to manually resolve these "merge conflicts" by deciding if you want to keep that part between <<<< and === or the part between === and >>>